eth/main.cpp

```diff
@@ -907,7 +904,7 @@ int main(int argc, char** argv)
 		catch (...)
 		{
 			cerr << "provided configuration is not well formatted" << endl;
-			cerr << "sample: " << endl << genesisInfo(eth::Network::Olympic) << endl;
+			cerr << "sample: " << endl << genesisInfo(eth::Network::Frontier) << endl;
```

Comment:
Isn't that string gigantic (it includes all the presale info)? Please use ropsten.